shared/domain/src/main/java/gov/va/vro/model/mas/MasCollectionAnnotation.java

```diff
@@ -0,0 +1,28 @@
+package gov.va.vro.model.mas;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.List;
+
+/** @author warren @Date 10/5/22 */
+@NoArgsConstructor
+@Getter
+@Setter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class MasCollectionAnnotation {
+  @JsonProperty("collectionsId")
```

Comment:
Is this some kinda of convention that we are always included @JsonProperty? No need for this is the property name and field name are the same.